### DISCUSSION TITLE
关于rife插帧

### DISCUSSION THREAD

**Discussion by stevejjla** [diana7127/mpv.net-DW#26] [Q&A] [answered]
这个定制版用了一段时间感觉还挺好用的
rife插帧的模型版本是v4,找了下新版的模型已经到了4.13,4.14了
尝试替换了一下新版的flownet.bin和flownet.param俩个文件会闪退,不知道有解决办法吗

**Comment by diana7127** [comment, marked as answer]
![PixPin_2024-01-25_20-20-07](https://github.com/diana7127/mpv.net-DW/assets/125502871/61aacdf1-d9c5-41b8-a3db-e2f05af646a4)
你看一下mpv_lazy的增强包，按照包内原来的命名和位置，复制粘贴到mpv.net-DW文件夹内

**Reply by stevejjla** [reply]
试了一下将librife.dll粘贴过去后,打开插帧不会闪退了,但实际上插帧没有运行,依然是24帧
又尝试了一下将portable_config内的vs里的MEMC_RIFE_STD.vpy也替换,还是无法运行插帧Orz
